api/src/main/java/org/opennms/integration/api/v1/info/Information.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.integration.api.v1.info;
+
+import java.util.Map;
+
+import org.opennms.integration.api.v1.annotations.Model;
+
+@Model
+public interface Information {
+
+    // Copied from: org.opennms.web.rest.v1.InfoDTO
+    // TODO: Patrick: discuss with core team if we need all these attributes in OIA?
+    DatetimeformatConfig getDatetimeformatConfig();
+    String getDisplayVersion();
+    String getVersion();
+    String getPackageName();
+    String getPackageDescription();
+    TicketerConfig getTicketerConfig();
+    Map<String,String> getServices();
```

Comment:
Not sure about this one. What does it even mean?

Reply:
I am not sure. The interface is a 1:1 copy of https://github.com/OpenNMS/opennms/blob/release-30.x/opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/InfoDTO.java

Reply:
Oh, this looks like something like mapping from daemon/service name to its current status (similar to what `bin/opennms -v status` would give). Feels ugly to have this in such an un-typed way. If there is currently no need for this and you don't feel eager to build some well-defined enum for this: I would vote for removing this. (Purely based on the fact that adding later on is way easier than altering).

Reply:
:+1:

Reply:
The frustrating thing is that it depends on how it's defined _in_ the `service-configuration.xml` file and we are unfortunately relying on convention for parsing all this stuff right now.

I don't see us changing how service config is set up any time soon though. Maybe it'd be good to actually hard fail if a service is seen that isn't understood and doesn't matched up to our more strictly-defined interface.

Reply:
> I don't see us changing how service config is set up any time soon though. Maybe it'd be good to actually hard fail if a service is seen that isn't understood and doesn't matched up to our more strictly-defined interface.

@RangerRick Do you see any need in adding this to the OIA api right now at all? Or would it be okay to not expose the field until we have a solution or some requirement?

Reply:
I think the idea was to expose exactly the same via rest. Therefor we would need it. Did I understand that right @RangerRick ?